10.0/apache/docker-entrypoint.sh

```diff
@@ -1,9 +1,45 @@
 #!/bin/bash
 set -e
 
-if [ ! -e '/var/www/html/version.php' ]; then
-    tar cf - --one-file-system -C /usr/src/nextcloud . | tar xf -
+# version_greater A B returns whether A > B
+function version_greater() {
+	[[ "$(printf '%s\n' "$@" | sort -V | head -n 1)" != "$1" ]];
+}
+
+installed_version="0.0.0~unknown"
+if [ -f /var/www/html/version.php ]; then
+    installed_version=$(php -r 'require "/var/www/html/version.php"; echo "$OC_VersionString";')
+fi
+image_version=$(php -r 'require "/usr/src/nextcloud/version.php"; echo "$OC_VersionString";')
+
+if version_greater "$installed_version" "$image_version"; then
+    echo "Downgrade not supported"
```

Comment:
Should probably be more specific. Something like 
```
echo "Can't start Nextcloud because the version of the data ($installed_version) is higher than the docker image version ($image_version) and downgrading is not supported. Are you sure you have pulled the newest image version?"
```